app/src/main/java/ar/com/wolox/android/training/ui/training/login/LoginFragment.java

```diff
@@ -60,13 +68,22 @@ public void init() {
             view.findViewById(R.id.btn_login).setOnClickListener(this);
             view.findViewById(R.id.btn_singup).setOnClickListener(this);
             view.findViewById(R.id.tyc_txt).setOnClickListener(this);
+
+            SignInButton signInButton = view.findViewById(R.id.btn_google_login);
+            signInButton.setSize(SignInButton.SIZE_STANDARD);
+            signInButton.setOnClickListener(this);
         }
 
         mProgressDialog = new ProgressDialog(getContext());
         mProgressDialog.setProgressStyle(ProgressDialog.STYLE_SPINNER);
         mProgressDialog.setCancelable(false);
         mProgressDialog.setCanceledOnTouchOutside(false);
 
+        GoogleSignInOptions gso = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
```

Comment:
You could (and you should I think, since you already have it) add this to the adapter.